libcxx/test/std/utilities/format/format.tuple/parse.pass.cpp

```diff
@@ -41,7 +41,7 @@ constexpr void test(StringViewT fmt, std::size_t offset) {
   static_assert(std::semiregular<decltype(formatter)>);
 
   std::same_as<typename StringViewT::iterator> auto it = formatter.parse(parse_ctx);
-  assert(it == fmt.end() - offset);
+  assert(std::to_address(it) == std::to_address(fmt.end()) - offset);
```

Comment:
Since this works for libc++ can you add a comment referring to the LWG issue you created.
(That should avoid people trying to "simplify the code.)
Searching for the original `assert` in the libc++ tests finds more occurrences. Can you fix these too?

Reply:
Can you also add a comment? Other than that if LGTM!

Reply:
yes please. I really want to discourage people in five years to "clean up" this code and break MSVC STL.